components/core/tools/scripts/db/init-db.py

```diff
@@ -18,30 +19,27 @@
 
 def main(argv):
     args_parser = argparse.ArgumentParser(description="Setup a global MySQL metadata database for CLP.")
-    args_parser.add_argument("--config-file", required=True, help="Metadata database basic config file.")
-    parsed_args = args_parser.parse_args(argv[1:])
-
-    config_file_path = parsed_args.config_file
+    args_parser.add_argument( "--db-host", default="127.0.0.1", help="Database host" )
+    args_parser.add_argument( "--db-port", type=int, default=3306,  help="Database port" )
+    args_parser.add_argument(  "--db-name",  default="clp-db", help="Database name" )
+    args_parser.add_argument( "--db-table-prefix", default="clp_", help="Database table prefix" )
 
-    with open(config_file_path, 'r') as f:
-        config = yaml.safe_load(f)
-    if config is None:
-        raise Exception(f"Unable to parse configuration from {config_file_path}.")
+    parsed_args = args_parser.parse_args(argv[1:])
 
-    required_keys = ["host", "port", "username", "password", "name"]
-    for key in required_keys:
-        if key not in config:
-            raise Exception(f"'{key}' missing from config file.")
+    host = parsed_args.db_host
+    port = parsed_args.db_port
+    db_name = parsed_args.db_name
+    table_prefix = parsed_args.db_table_prefix
 
-    host = config["host"]
-    port = config["port"]
-    username = config["username"]
-    password = config["password"]
-    db_name = config["name"]
-    table_prefix = config["table_prefix"]
+    try:
+        username = os.environ["CLP_DB_USER"]
+        password = os.environ["CLP_DB_PASS"]
+    except KeyError as e:
+        key = e.args[0]
+        raise Exception(f"'{key}' is missing from environment variables.")
 
     try:
-        mysql_conn = mariadb.connect(host=host, port=port, username=username, password=password)
+        mysql_conn = mariadb.connect(host=host, port=port, username=username, password=password, )
```

Comment:
```suggestion
        mysql_conn = mariadb.connect(host=host, port=port, username=username, password=password)
```
I don't think we have coding guidelines on this, but [PEP8](https://peps.python.org/pep-0008/#when-to-use-trailing-commas) recommends against using trailing commas for function calls unless each argument is on a different line.